assets/app/scripts/app.js

```diff
@@ -194,6 +195,14 @@ angular
         templateUrl: 'views/browse/service.html',
         controller: 'ServiceController'
       })
+      .when('/project/:project/browse/storage', {
+        templateUrl: 'views/storage.html',
+        controller: 'StorageController'
+      })
+      .when('/project/:project/browse/persistentvolumeclaims/:pvc', {
```

Comment:
Wondering if it's too late to use short names in our URLs: `/browse/pvc`, `/attachPVC`? Not consistent with what we already have unfortunately.